parquet-avro/src/main/java/parquet/avro/AvroIndexedRecordConverter.java

```diff
@@ -36,7 +34,6 @@
 import parquet.io.api.PrimitiveConverter;
 import parquet.schema.GroupType;
 import parquet.schema.MessageType;
-import parquet.schema.OriginalType;
 import parquet.schema.Type;
 
 class AvroIndexedRecordConverter<T extends IndexedRecord> extends GroupConverter {
```

Comment:
Can you add a bit of documentation to explain the salient differences between this class and AvroRecordConverter? It's quite hard to tell just by looking.